doc/source/conf.py

```diff
@@ -28,8 +28,21 @@
     'IPython.sphinxext.ipython_console_highlighting',
     'IPython.sphinxext.ipython_directive',
     'sphinx.ext.imgmath',
+    'sphinx_affiliates',
 ]
 
+# Canonical URL (including the '/') so searching from numpy docs finds these
+affiliate_options = {
+    'canonical_url': "https://numpy.org/devdocs",
```

Comment:
This will mean that after publishing 1.20 as stable, the canonical url for the search index will still point to dev-docs. Is that fine?

Reply:
I guess the URL should be part of the __other__ site's code. That design actually makes things simpler: there is no need for a `canonical_url`. Need to think about it a bit.